index.html

```diff
@@ -0,0 +1,158 @@
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+  <!-- the Home page -->
+  <section id="home">
+    <div class="main-content">
+      <div class="headline">
+        <h2 class="heading1">"Hello! Sharing World"</h2>
+        <h1 class="main-heading">HopeGrower Gold Global Conference 2023</h1>
+      </div>
+      <p class="main-detail">A joyful celebration believing in the openness and sharing, believing in "Food is the New
+        Gold" creating a positive
+        change with people from over 100 countries is taking place in November, Pakistan</p>
+      <div class="date-venue">
+        <h2 class="dates">2023.11.14(THU) ~ 15(FRI)</h2>
+        <p class="venue">@ National Museum of Pakistan, Art Center Iqbal & more</p>
+      </div>
+    </div>
+  </section>
+  <!-- the Program page -->
+  <section id="program">
+    <div>
+      <h3 class="program-headline">Main Program</h3>
+      <div class="underline"></div>
+    </div>
+    <div class="program-container">
+      <div class="main-program">
+        <div><img src="/img/program_icon_01.png" alt=""></div>
+        <h3 class="program-subheading">
+          Lecture
+        </h3>
+        <p class="program-info">
+          Listen to the speakers from various countries about the message of farming and food.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_02.png" alt=""></div>
+        <h3 class="program-subheading">
+          HG Exhibition
+        </h3>
+        <p class="program-info">
+          Appreciate various creations applying CC license of artists, organized from Art Center Iqbal
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_03.png" alt=""></div>
+        <h3 class="program-subheading">
+          Forum
+        </h3>
+        <p class="program-info">
+          Have the time to share your thoughts and opinions with experts for each topic.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_04.png" alt=""></div>
+        <h3 class="program-subheading">
+          Workshop
+        </h3>
+        <p class="program-info">
+          Try creating your own work using open source license rather than just watching at it.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_05.png" alt=""></div>
+        <h3 class="program-subheading">
+          HG Ignite
+        </h3>
+        <p class="program-info">
+          Get opportunities to network with HG affiliates around the world, also after the summit.
+        </p>
+      </div>
+    </div>
+    <button type="button" class="join-btn">Join HG Global Summit 2023</button>
+  </section>
+  <!-- the Speakers page -->
+  <section><h3 class="speaker-headline">Featured Speakers</h3></section>
```

Comment:
- [ ] kindly add some padding to the speaker heading top and bottom. also please add the font size 
- [ ] like in the original design, add margin left and right to the speaker section.
- [ ] kindly add the red line under the partner section heading. nice work.
your design
![Screenshot (1078)_LI](https://github.com/Mike111222/CapstoneProject/assets/65298048/d0f01f87-8380-467c-968e-0e5b13b6993f)
original
![Screenshot (1078)](https://github.com/Mike111222/CapstoneProject/assets/65298048/611e25af-818c-43dd-9f92-973674aa8e7b)